ci-operator/config/openshift/oc/openshift-oc-master.yaml

```diff
@@ -60,17 +61,35 @@ resources:
     requests:
       cpu: "3"
       memory: 6Gi
-rpm_build_commands: make rpm-build
-rpm_build_location: _output/rpms/
+  'build-rpms-from-tar':
+    limits:
+      memory: 1Gi
+    requests:
+      cpu: "100m"
+      memory: 100Mi
+rpm_build_commands: |-
```

Comment:
@stevekuznetsov btw. the need for `|-` instead of `|` is a bug in ci-operator - it renders:
```
; ln -s $( pwd )/_rpmbuild/RPMS/x86_64/ /srv/repo' failed with exit code 2
/bin/bash: -c: line 9: syntax error near unexpected token `;'
```
took me a while to figure it out